pkg/version/version.go

```diff
@@ -138,6 +138,30 @@ func GetExistingOperatorVersion(namespace string) (string, error) {
 	return version, nil
 }
 
+func GetExistingEtcdOperatorVersion(namespace string) (string, error) {
+	if namespace == "" {
+		namespace = "storageos-etcd"
+	}
+	config, err := pluginutils.NewClientConfig()
+	if err != nil {
+		return "", err
+	}
+
+	clientset, err := pluginutils.GetClientsetFromConfig(config)
+	if err != nil {
+		return "", errors.Wrap(err, consts.ErrUnableToContructClientFromConfig)
+	}
+	etcdDeployment, err := clientset.AppsV1().Deployments(namespace).Get(context.TODO(), "storageos-etcd-controller-manager", metav1.GetOptions{})
```

Comment:
`storageos-etcd-controller-manager` may need a constant, because it is used in multiple places